cmd/e2e_test.go

```diff
@@ -31,12 +33,14 @@ func init() {
 	rootDir, _ := os.Getwd()
 	flag.StringVar(&webhookConfigPath, "webhookConfigPath", "", "path to webhook config file")
 	flag.StringVar(&demoSuitesPath, "config-suites", fmt.Sprintf(rootDir+"/tests/e2e-demos/config/default.yaml"), "path to e2e demo suites definition")
+	flag.BoolVar(&generateRPPreprocReport, "generate-rppreproc-report", false, "Generate report and folders for RP Preproc")
 }
 
 func TestE2E(t *testing.T) {
 	klog.Info("Starting Red Hat App Studio e2e tests...")
 	// Setting viper configurations in cache
 	viper.Set("config-suites", demoSuitesPath)
+	viper.Set("generate-rppreproc-report", generateRPPreprocReport)
```

Comment:
Is this necessary? I don't see anywhere in the code that you would be obtaining the value from Viper.

Reply:
Deleted